src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -188,10 +187,11 @@ static ucs_status_t uct_gdr_copy_mem_reg(uct_md_h uct_md, void *address, size_t
         return UCS_ERR_NO_MEMORY;
     }
 
-    reg_size = (length + GPU_PAGE_SIZE - 1) & GPU_PAGE_MASK;
-    ptr = (void *) ((uintptr_t)address & GPU_PAGE_MASK);
+    start = ucs_align_down_pow2_ptr(address, GPU_PAGE_SIZE);
```

Comment:
Generic question - is this safe to use compile time GPU_PAGE_SIZE ?

Reply:
@shamisp Do you mean this in the context of `#if HAVE_CUDA`?
If so, GPU_PAGE_SIZE is defined in [gdrapi.h](https://github.com/NVIDIA/gdrcopy/blob/master/gdrapi.h) and gets included `#if HAVE_CUDA`. It should be safe, correct?

Reply:
This is NVIDIA's gdr_copy constant https://github.com/NVIDIA/gdrcopy/blob/master/gdrapi.h#L35
Looks like it is safe for current nvidia architectures.

Reply:
I just wanted to make sure that it is the same value (64K) across all architectures.